lib/js/components/Tile/Tile.vue

```diff
@@ -173,26 +173,25 @@ $tile-colors: (
 	}
 
 	&__center {
-		@include textBold;
-
 		display: flex;
 		flex-direction: column;
 		flex-grow: 1;
 	}
 
 	&__eyebrowText {
-		@include textS;
+		@include InfoMBold();
 
 		&.-uppercase {
-			text-transform: uppercase;
+      /* @todo Add InfoMBoldUppercase when it appears in JSON */
```

Comment:
ticket or do we want to address it in this PR?

Reply:
This PR. This was reported to UX this morning and we're waiting to see what they decide.